conans/client/rest/uploader_downloader.py

```diff
@@ -122,7 +123,7 @@ def __iter__(self):
         if self.output and self.output.is_terminal:
             progress_bar = tqdm(total=self.totalsize, unit='B', unit_scale=True,
                                 unit_divisor=1024, desc="Uploading {}".format(self.file_name),
-                                leave=False, dynamic_ncols=True, ascii=False)
+                                leave=True, dynamic_ncols=False, ascii=True)
```

Comment:
```suggestion
                                leave=True, dynamic_ncols=False, ascii=True, file=self.output)
```
Line 139: `self.output.rewrite_line("{} [done]".format(progress_bar.desc))` 
should be removed

Line 54: `self.output.info("")` should be removed too to avoid extra lines 
but left for `not self.output.is_terminal`